pkgs/applications/video/youtube-tui/default.nix

```
@@ -17,21 +17,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "youtube-tui";
-  version = "0.8.0";
+  version = "0.8.1";
 
   src = fetchFromGitHub {
     owner = "Siriusmart";
     repo = "youtube-tui";
     rev = "v${version}";
```

Might be a good time to use `tag` here instead

```suggestion
    tag = "v${version}";
```

---

Wow when was that added? Nixpkgs changes so fast :eyes:

---

> Wow when was that added? Nixpkgs changes so fast 👀

Quite recently, in #362692

---

done.